<launch>
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen" />
  <node name="alert_player" pkg="alert_player" type="alert_player.py" output="screen">
    <rosparam command="delete" param=""/>
    <param name="sleep_time" value="1" />
    <param name="volume" value="1.0" />
    <param name="topic/alert_play" value="/alert/play" />
    <param name="alert_base_path" value="/opt/ros_ws/src/alert_player/alert_data/" />
  </node>
</launch>
